ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -321,20 +294,16 @@ public void onAdLoad(String id) {
         public void onError(String id, VungleException exception) {
           Log.d(TAG, "Ad load failed:" + VungleBannerAdapter.this);
           VungleManager.getInstance().removeActiveBannerAd(mPlacementId);
-          VungleListener listener = mVungleListener.get();
-          if (mPendingRequestBanner && listener != null) {
-            listener.onAdFailedToLoad(AdRequest.ERROR_CODE_NO_FILL);
-          }
+          mVungleBannerListener.onAdFailedToLoad(AdRequest.ERROR_CODE_NO_FILL);
```

Comment:
shouldn't we check for mPendingRequestBanner here?

Reply:
done.